run/restic2john.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
```

Comment:
Please add copyright and license in here, perhaps similar to what's in `restic_fmt_plug.c`.